test/studies/shootout/submitted/regexredux3.chpl

```diff
@@ -31,7 +31,7 @@ proc main(args: [] string) {
   const initLen = data.size;
 
   // remove newlines
-  data = compile(b">.*\n|\n").sub(b"", data);
```

Comment:
Sorry, meant to indicate that this test also shouldn't have been updated, but I could have made that more clear

Reply:
This is on me. I know that we don't change `submitted` directory. Even after your warning for the first one, I still missed this one.

Thanks for checking after the merge. Sorry about being distracted and causing churn here. I'll fix.

Reply:
It happens, no worries :)